tip-wlan/example-values/aws-internal/values.yml

```diff
@@ -0,0 +1,131 @@
+shared:
+  service:
+    srv-https-annotations: &srv-https-annotations
+      kubernetes.io/ingress.class: alb
+      alb.ingress.kubernetes.io/scheme: internet-facing
```

Comment:
Mmh, think this should be set to `internal` as well, or?

Reply:
True, missed that somehow

Reply:
Fixed